Mathlib/Analysis/Calculus/VectorField.lean

```diff
@@ -179,6 +179,26 @@ lemma lieBracket_add_left (hV : DifferentiableAt 𝕜 V x) (hV₁ : Differentiab
   rw [fderiv_add hV hV₁, ContinuousLinearMap.add_apply]
   abel
 
+/-- We have `[0, W] = 0` for all vector fields `W`: this depends on the junk value 0
+if `W` is not differentiable. Version within a set. -/
+@[simp]
+lemma lieBracketWithin_zero_left : lieBracketWithin 𝕜 0 W s x = 0 := by simp [lieBracketWithin]
```

Comment:
```suggestion
lemma lieBracketWithin_zero_left : lieBracketWithin 𝕜 0 W s = 0 := by ext; simp [lieBracketWithin]
```
This will make the simp lemma more powerful. Same thing below. I expect this will help in the proof of mlieBracketWithin_zero_left, for instance.